frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx

```diff
@@ -297,197 +297,195 @@ export const HWCRComplaintAssessment: FC = () => {
+                  options={assessmentTypeList}
+                  onCheckboxChange={handleAssessmentTypesChange}
+                  checkedValues={selectedAssessmentTypes}
+                ></ValidationCheckboxGroup>
               </div>
             </div>
-            <div className="comp-details-edit-container">
-              <div className="comp-details-edit-column">
-                <div
-                  id="outcome-officer-div"
-                  className="assessment-details-label-input-pair"
-                >
-                  <label htmlFor="outcome-officer">Officer</label>
-                  {editable ? (
-                    <CompSelect
-                      id="outcome-officer"
-                      className="comp-details-input"
-                      classNamePrefix="comp-select"
-                      options={assignableOfficers}
-                      enableValidation={true}
-                      errorMessage={officerErrorMessage}
-                      value={selectedOfficer}
-                      placeholder="Select "
-                      onChange={(officer: any) => setSelectedOfficer(officer)}
-                    />
-                  ) : (
-                    <div
-                      data-initials-sm={getAvatarInitials(selectedOfficer?.label ?? "")}
-                      className="comp-orange-avatar-sm comp-details-inner-content"
-                    >
-                      <span
-                        id="comp-review-required-officer"
-                        className="comp-padding-left-xs"
-                      >
-                        {selectedOfficer?.label ?? ""}
-                      </span>
-                    </div>
-                  )}
-                </div>
-              </div>
-              <div className="comp-details-edit-column comp-details-right-column">
-                <div
-                  id="complaint-outcome-date-div"
-                  className="assessment-details-label-input-pair"
-                >
-                  <label htmlFor="complaint-outcome-date">Date</label>
-                  {editable ? (
-                    <ValidationDatePicker
-                      id="complaint-outcome-date"
-                      selectedDate={selectedDate}
-                      onChange={handleDateChange}
-                      placeholder="Select date"
-                      className="comp-details-edit-calendar-input" // Adjust class as needed
-                      classNamePrefix="comp-select" // Adjust class as needed
-                      errMsg={assessmentDateErrorMessage} // Pass error message if any
-                      maxDate={new Date()}
-                    />
-                  ) : (
-                    formatDate(`${selectedDate}`)
-                  )}
-                </div>
-              </div>
+          </div>
+          <div
+            id="justification-div"
+            className="comp-details-form-row"
+          >
+            <label
```

Comment:
The Justification field should only show when Action Required is set to no.

Reply:
Updated to provide accurate display of form field here.